freqtrade/templates/strategy_analysis_example.ipynb

```diff
@@ -23,22 +23,36 @@
    "metadata": {},
    "outputs": [],
    "source": [
+    "import os\n",
     "from pathlib import Path\n",
+    "\n",
+    "# Change current working directory from `somedir/freqtrade/user_data/notebooks` to project root `somedir/freqtrade`, so relative paths remain consistent.\n",
+    "if not Path(\"LICENSE\").is_file():\n",
+    "    os.chdir(\"../../\")\n",
```

Comment:
I'm not sure why this is necessary - i think it hints at a wrong jupyter startup location rather than something that should be done in the notebook.

the assumption is that jupyter (or jupyter lab) is started at the repository's root - which contains `user_data/` - in which case the above is wrong (or at least unnecessary).

i don't really see why we should force users to change directory (to a potentially non-existing directory).
if a non-standard startup location is chosen, i'd assume that the user also knows how to change directory.

This code is also failing if it's starting from within user_data - as then it'd go up one level too much. As such - i don't currently see this as something we should be adding to the notebook example.

Reply:
This is already somewhere in the freqtrade document, but not convenient, https://www.freqtrade.io/en/stable/data-analysis/#change-directory-to-root

> the assumption is that jupyter (or jupyter lab) is started at the repository's root

As I can see, Jupyter start at the notebook file location by default, which is `freqtrade/user_data/notebooks`, not the repository' root.